productinfo/productinfo.go

```diff
@@ -27,6 +27,12 @@ const (
 
 	// PriceKeyTemplate format for generating price cache keys
 	PriceKeyTemplate = "/banzaicloud.com/recommender/%s/%s/prices/%s"
+
+	// ZoneKeyTemplate format for generating zone cache keys
+	ZoneKeyTemplate = "/banzaicloud.com/recommender/%s/%s/zones/"
```

Comment:
Can `/banzaicloud.com/recommender` change in the future? If so make this configurable

Reply:
this has no impact if the domain changes ...

Reply:
fair enough